pipeline/Makefile

```diff
@@ -0,0 +1,109 @@
+.DEFAULT_GOAL := help
+
+CONFIG_PATH := brca_pipeline_cfg.mk
+include $(CONFIG_PATH)
+
+checkout: ## checking out code base
+	[ -d $(CODE_BASE) ] || git clone $(BRCA_GIT_REPO) $(CODE_BASE)
+	cd $(CODE_BASE) && git checkout $(GIT_COMMIT)
+
+build-docker: ## build main pipeline docker image
+	cd $(CODE_BASE)/pipeline/docker && ./build_docker_image.sh $(GIT_COMMIT) $(RELEASE_TAG) || { echo "$(date) Something wrong with docker image creation"; exit 1; }
+
+download-resources: ## download resources files
+	$(CODE_BASE)/pipeline/download_resources_files.sh $(RESOURCES_DIR) >> "$(LOG_DIR)/download_resources_$(RELEASE_TAG).log" 2>&1 || { echo "Downloading resource files failed"; exit 1; }
+
+
+start-local-uta: ## starting local uta docker container
+	[ `docker ps -f name=$(UTA_CONTAINER) | wc -l` -gt 1 ] || docker run -dit --name $(UTA_CONTAINER) -p $(UTA_PORT):5432 $(UTA_DOCKER_IMAGE)
+
+.ONESHELL:
```

Comment:
Another OS X note: the default version of `make` shipped with OS X (at least on my version, 10.13.6) doesn't support `.ONESHELL`. Upgrading to 3.82 fixes the problem.